src/components/application_manager/include/application_manager/application.h

```diff
@@ -51,6 +51,7 @@
 #include "protocol_handler/protocol_handler.h"
 #include "smart_objects/smart_object.h"
 #include "utils/macro.h"
+#include "utils/semantic_version.h"
```

Comment:
why this PR include changes for "Mobile API Versioning Implementation"?

Reply:
Changes are merged into develop now, so they aren't present in this PR anymore. In any case, since existing parameters are being changed, Mobile API Versioning is needed.